scripts/travis/external_build.sh

```diff
@@ -74,6 +74,7 @@ fi
 set +e
 # remove if it's already there, so the new build would replace it.
 aws s3 rm ${BUILD_COMPLETE_PATH} ${NO_SIGN_REQUEST}
+aws s3 rm ${BUILD_LOG_PATH}-1 ${NO_SIGN_REQUEST}
```

Comment:
this file isn't really related to this PR. It's part of a separate unrelated build-PR that I've been working on.